src/js/boot.js.tpl

```diff
@@ -15,14 +15,16 @@ define([], function() {
             // Loading message while we fetch JS / CSS
             el.innerHTML = '<div style="font-size: 24px; text-align: center; padding: 72px 0; font-family: \'Guardian Egyptian Web\',Georgia,serif;">Loading…</div>';
 
-            config = {
-                'assetPath': '<%= assetPath %>',
+            const extraConfig = {
+                'assetPath': '',
```

Comment:
As far as I can tell, `config` is injected by [frontend](https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/standard/main.js#L132-L140)(namely line [137](https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/standard/main.js#L137))

Before, we were overriding it 😊. Now we extend it if its set, or set it to `extraConfig` if its not set.